cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -108,20 +106,30 @@ def publisher(self, key, value):
     return clean_val("b", value, str, req=True)
 
 
-@model.over("_children", "(^362__)")
-def children_records(self, key, value):
-    """Translates fields related to children record types."""
-    _migration = self["_migration"]
-    _electronic_items = _migration.get("electronic_items", [])
-    if key == "362__":
-        _electronic_items.append({"subscription": clean_val("a", value, str)})
-    _migration.update(
-        {
-            "electronic_items": _electronic_items,
-        }
-    )
+@model.over("electronic_volumes_description", "(^362__)")
+def electronic_volumes_description(self, key, value):
+    """Translates electronic volumes description field."""
+    description = clean_val("a", value, str, req=True)
+    if description.endswith('-'):
```

Comment:
I think this hyphen is there on purpose, could you check with the library ?

Reply:
You are right, it's on purpose and we should keep them, library answer:
"it is important to keep it, those fields are meant to describe a time period, when there is no 2nd volume/year it is because the collection or publication continues till now"